src/crimson/net/ProtocolV2.cc

```diff
@@ -786,7 +791,7 @@ void ProtocolV2::execute_connecting()
         }).then([this] {
           return banner_exchange();
         }).then([this] (entity_type_t _peer_type,
-                        entity_addr_t _peer_addr) {
+                        entity_addr_t _my_addr_from_peer) {
```

Comment:
i'd suggest avoid having refactory only changes along with functional changes in a single commit unless you have to.

Reply:
splitted into 4 clean-up commits.